pages/home.js

```diff
@@ -49,7 +49,7 @@ const HomePage = () => {
 
 export const getServerSideProps = async ({ req, res }) => {
   const cookies = cookie.parse((req && req.headers.cookie) || '');
-  const redirectToDashboard = req.url === '/' && cookies.rootRedirect === 'dashboard';
+  const redirectToDashboard = req.url === '/' && cookies.rootRedirect;
```

Comment:
In this case, shouldn't we redirect to whatever `rootRedirect` holds?

Reply:
Yeah, ideally that would be the case, however the value was not being set properly even though the cookie was there.

Anyway, will close this branch in favour of https://github.com/opencollective/opencollective-frontend/pull/9536